### PR TITLE
New Enabling Add-On Task

### PR DESCRIPTION
**Pull request by Pothulapati**
This is a new task which shows users on how to enable the tracing add-on.
Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

**Comment by grampelberg** [comment]
@cpretzer mind giving this a runthrough?

**Comment by cpretzer** [comment]
Yep, I'll take a look

**Comment by grampelberg** [comment]
@cpretzer did you get a chance to run through this yet?

**Comment by cpretzer** [comment]
Not yet. I'll add it to my queue for today

File: linkerd.io/content/2/tasks/enabling-addons.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
++++
+title = "Enabling Add-Ons"
+description = "Great Out-of-the-box experience with various components that integrate well with Linkerd"
+aliases = [
+  "/2/add-ons/",
+]
++++
+
+Linkerd can also be installed with a number of Add-On's, allowing users to get a
```
Comment (cpretzer) [inline linkerd.io/content/2/tasks/enabling-addons.md]: Is there a description or a list of "add-on's" that we can link to?

Comment (Pothulapati) [reply linkerd.io/content/2/tasks/enabling-addons.md]: I wrote a statement about this at https://github.com/linkerd/website/pull/686/files#diff-21202a0d17098d0883ac4e6e75aaead1R25

I can link the same here too

Comment (Pothulapati) [reply linkerd.io/content/2/tasks/enabling-addons.md]: Done

Comment (cpretzer) [reply linkerd.io/content/2/tasks/enabling-addons.md]: awesome, thanks!

**Pull request merged**